src/lib/popover/popover.spec.ts

```diff
@@ -350,6 +355,79 @@ describe('SatPopover', () => {
 
   });
 
+  describe('positioning', () => {
+    let fixture: ComponentFixture<PositioningTestComponent>;
+    let comp:    PositioningTestComponent;
+    let overlayContainerElement: HTMLElement;
+
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [
+          SatPopoverModule,
+          NoopAnimationsModule,
+        ],
+        declarations: [PositioningTestComponent],
+        providers: [
+          {provide: OverlayContainer, useFactory: overlayContainerFactory}
+        ]
+      });
+
+      fixture = TestBed.createComponent(PositioningTestComponent);
+      comp = fixture.componentInstance;
+
+      overlayContainerElement = fixture.debugElement.injector.get(OverlayContainer)
+        .getContainerElement();
+    });
+
+    afterEach(() => {
+      document.body.removeChild(overlayContainerElement);
+    });
+
+    it('should reconstruct the overlay when positions are updated', fakeAsync(() => {
```

Comment:
Need to add a baseline test, showing that overlayRefs are *not* recreated on typical subsequent openings